### PR TITLE
lesson-1-3

### PR DESCRIPTION
**Pull request by Armanchi**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,11 +2,14 @@ import React from 'react';
...
 import AddTodoForm from './AddTodoForm';
 
-function App() {
+const App = () => {
+  const [newTodo, setNewTodo] = React.useState(''); 
   return (
     <div>
       <h1>Todo List</h1>
+      <AddTodoForm onAddTodo = {setNewTodo} />
       <AddTodoForm />
```
Comment (kramduckner) [inline src/App.js]: Let's remove the AddTodoForm component that doesn't have the onAddTodo prop!

File: src/TodoListItem.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const TodoListItem = (props) => {
+    return (
+        <span>
```
Comment (kramduckner) [inline src/TodoListItem.js]: I like keeping the html as simple as possible by only using html tags that are needed! I think it makes the code easier to maintain over time. Would you mind removing the span element?

**Review by kramduckner — CHANGES_REQUESTED** [review]
Great work @Armanchi ! Just a couple of small requests.

File: src/App.js (hunk 1 above)
Comment (Armanchi) [reply src/App.js]: Okie dokie! Just made changes.

**Review by kramduckner — APPROVED** [review]
Thanks! Approved. Feel free to merge

**Pull request merged**